sdks/java/core/src/main/java/org/apache/beam/sdk/io/BigQueryIO.java

```diff
@@ -201,8 +201,9 @@
  *
  * <p>See {@link BigQueryIO.Write} for details on how to specify if a write should
  * append to an existing table, replace the table, or verify that the table is
- * empty. Note that the dataset being written to must already exist. Write
- * dispositions are not supported in streaming mode.
+ * empty. Note that the dataset being written to must already exist. In streaming mode, the Write
+ * disposition must be {@link WriteDisposition#WRITE_EMPTY empty} or
```

Comment:
ping @dhalperi, @dpmills 

For context: the [existing validation in BigQueryIO](https://github.com/apache/incubator-beam/pull/451/files#diff-33d4295bebecbdb7d6718630e710074aL1684) disallows using `WriteDisposition.WRITE_TRUNCATE` with streaming. @bjchambers is asking if we should also disallow `WRITE_CREATE`.